packages/office-ui-fabric-react/src/components/DetailsList/DetailsRow.styles.ts

```diff
@@ -0,0 +1,426 @@
+import { IDetailsRowStyleProps, IDetailsRowStyles } from './DetailsRow.types';
+import {
+  getGlobalClassNames,
+  getFocusStyle,
+  HighContrastSelector,
+  FontSizes,
+  AnimationClassNames
+} from '../../Styling';
+import { IStyleBaseArray, IStyle } from '../../../../merge-styles/lib/IStyle';
+
+const GlobalClassNames = {
+  root: 'ms-DetailsRow',
+  compact: 'ms-DetailsList--Compact',
+  cell: 'ms-DetailsRow-cell',
+  cellCheck: 'ms-DetailsRow-cellCheck',
+  cellMeasurer: 'ms-DetailsRow-cellMeasurer',
+  listCellFirstChild: 'ms-List-cell:first-child',
+  isFocusable: "data-is-focusable='true'",
+  isContentUnselectable: 'is-contentUnselectable',
+  isSelected: 'is-selected',
+  isCheckVisible: 'is-check-visible',
+  fields: 'ms-DetailsRow-fields'
+};
+
+// Constant values
+let values = {
+  rowHeight: 42,
+  compactRowHeight: 32,
+  rowVerticalPadding: 11,
+  rowHorizontalPadding: 8,
+  compactRowVerticalPadding: 6,
+  isPaddedMargin: 24,
+  rowShimmerLineHeight: 7,
+  rowShimmerIconPlaceholderHeight: 16,
+  rowShimmerHorizontalBorder: 0,
+  rowShimmerVerticalBorder: 0,
+  compactRowShimmerVerticalBorder: 0
+};
+
+// Computed Values
+values = {
+  ...values,
+  ...{
+    rowShimmerHorizontalBorder: values.rowHorizontalPadding,
+    rowShimmerVerticalBorder: (values.rowHeight - values.rowShimmerLineHeight) / 2,
+    compactRowShimmerVerticalBorder: (values.compactRowHeight - values.rowShimmerLineHeight) / 2
+  }
+};
+
+export const getStyles = (props: IDetailsRowStyleProps): IDetailsRowStyles => {
+  const {
+    theme,
+    isSelected,
+    canSelect,
+    checkClassNames,
+    droppingClassName,
+    anySelected,
+    isCheckVisible,
+    checkboxCellClassName,
+    compact,
+    className
+  } = props;
+
+  const {
+    neutralPrimary,
+    neutralSecondaryAlt,
+    white,
+    neutralSecondary,
+    neutralLighter,
+    neutralLight,
+    neutralDark,
+    neutralQuaternaryAlt,
+    black,
+    themePrimary
+  } = theme.palette;
+
+  const { compactRowHeight, compactRowVerticalPadding, rowHorizontalPadding } = values;
+  const classNames = getGlobalClassNames(GlobalClassNames, theme);
+
+  const colors = {
+    // Default
+    defaultHeaderTextColor: neutralPrimary,
+    defaultMetaTextColor: neutralSecondaryAlt,
```

Comment:
neutralSecondaryAlt has been deprecated, can you spare the cycles to swap this to something else appropriate?
https://github.com/OfficeDev/office-ui-fabric-react/issues/5194

Reply:
which one should it be instead?